content/en/docs/concepts/containers/container-lifecycle-hooks.md

```diff
@@ -75,14 +75,14 @@ execution, the Pod's phase will be `Terminating` and remain there until the Pod
 both the `PreStop` hook to execute and for the Container to stop normally. If, for example,
 `terminationGracePeriodSeconds` is 60, and the hook takes 55 seconds to complete, and the Container
 takes 10 seconds to stop normally after receiving the signal, then the Container will be killed
-before it can stop normally, since `terminationGracePeriodSeconds` is less than the total time
+before it can stop normally since `terminationGracePeriodSeconds` is less than the total time
```

Comment:
I'm not sure if removing the `,` here is the right choice. Could you please let me know your reasoning?